src/java/org/apache/cassandra/db/compaction/AbstractCompactionTask.java

```diff
@@ -143,7 +143,7 @@ public int execute()
         }
     }
 
-    private Throwable cleanup(Throwable err)
+    public Throwable cleanup(Throwable err)
```

Comment:
I am not sure this change is good enough to provide the functionality of CNDB-2214. See my comment https://github.com/riptano/cndb/pull/4553/files#r855928353

Reply:
Ok, got it now, although I still don't quite get the purpose of calling `setOpObserver` here ...